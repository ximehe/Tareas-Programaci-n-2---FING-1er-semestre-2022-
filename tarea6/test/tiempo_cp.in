tiempo-cp 110000 1 2
Fin