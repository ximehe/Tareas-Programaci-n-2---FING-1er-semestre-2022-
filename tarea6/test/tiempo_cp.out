1>Bien.
2>Fin.
